packages/zoe/test/unitTests/zcf/zcf.test.js

```diff
@@ -296,7 +297,8 @@ test(`zcf.makeInvitation - non-string description`, async t => {
   // https://github.com/Agoric/agoric-sdk/issues/1704
   // @ts-expect-error deliberate invalid arguments for testing
   t.throws(() => zcf.makeInvitation(() => {}, { something: 'a' }), {
-    message: /invitations must have a description string: .*/,
+    message:
+      'In "makeInvitation" method of (zcf): arg 1: copyRecord {"something":"a"} - Must be a string',
```

Comment:
You asked for feedback on the error messages. In the absence of custom messages, these full path descriptions are very helpful. It is some loss that the message says nothing about the "a description string" but the interface doesn't know the name of positional parameters.

I don't think it worthwhile to add that feature, but for brainstorming,
```
  makeInvitation: M.call(M.raw(), M.named('description', M.string()))
```
Though we'd want to ensure the string in that name matches the param name in code, and that would take coding up a new lint rule.

Reply:
Interesting idea, thanks. Resolving this conversation here though.